modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -35,13 +35,6 @@ services can cause you to run out of resources.
 Deployments of later versions of {rh-openstack} that have the OVN Octavia driver disabled also
```

Comment:
This and paragraph above should probably be reworded too - "versions before 16" basically means 13.

Reply:
Seems to me that this whole section might be folded into the "general limitations" list. What do you think?